Chiazokam/Politico
# #163356189 create the get all offices endpoint

#### What does this PR do?
This PR tests and creates the GET all offices endpoint

#### Description of Task to be completed?
- write tests for all the functions
- create the endpoint

#### How should this be manually tested?
- Clone the repo - `https://github.com/Chiazokam/Politico`
- Enter into the project directory - `cd Politico`
- switch to this branch - `checkout ft-datastructures-get-all-offices-163356189`
- Start the server `npm run start:dev`
- Open `Postman`
- On `postman` Make a `GET` request to `http://localhost:3000/api/v1/offices `
- You should receive a successful response

#### Any background context you want to provide?
Nil
#### What are the relevant pivotal tracker stories?
[#163356189](https://www.pivotaltracker.com/story/show/163356189)

####  Images

![image](https://user-images.githubusercontent.com/26940294/51779080-73a11380-2105-11e9-8302-86a7b0c68e82.png)
